recipe/build.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+#
+source activate "${CONDA_DEFAULT_ENV}"
+
 if [ $(uname) == Darwin ]; then
     export LDFLAGS="-headerpad_max_install_names"
```

Comment:
So, this is overriding the `LDFLAGS` set by the `toolchain`. Also, we add this to `LDFLAGS` anyways. Maybe we can try dropping it.